code/modules/research/designs/medical_designs.dm

```diff
@@ -62,6 +62,16 @@
 	build_path = /obj/item/handheld_defibrillator
 	category = list("Medical")
 
+/datum/design/holostretcher
+	name = "Holo Strecher"
+	desc = "A hardlight projector for transporting patients."
+	id = "holo_stretcher"
+	req_tech = list("biotech" = 5, "magnets" = 4, "powerstorage" = 4)
+	build_path = /obj/item/roller/holo
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1000, MAT_SILVER = 500, MAT_GLASS = 500, MAT_DIAMOND = 200)
```

Comment:
Needing diamond might be a bit too expensive?
I don't expect people to want to print these anyway, making it cost lots of resources is probably not needed.

But then again, i'm not a science player, so i'm not quite sure if 200 diamond is "a lot"...

Reply:
200 is not a lot, and upgrades reduce this further